setup.cfg

```diff
@@ -28,11 +28,49 @@ keywords = TensorFlow, phasespace, HEP
 
 [options]
 packages = find:
+install_requires =
+    tensorflow>=2.3,<2.6
+    tensorflow_probability>=0.11,<0.13
 python_requires = >=3.6
 include_package_data = True
 testpaths = tests
 zip_safe = False
 
+[options.extras_require]
+test =
+    awkward
+    coverage
+    flaky
+    matplotlib
+    numpy
+    pytest
+    pytest-cov
+    pytest-xdist
+    scipy
+    uproot
+    wget
+dev =
+    %(test)s
+    bumpversion
+    coverage
+    flake8
+    flaky
+    matplotlib
+    numpy
+    pip>=18.1
+    pytest
```

Comment:
We should deduplicate, so remove the pytest- ones, wget etc. right?

And uproot has to be in the tests as well, can you move it up?

Reply:
Ah yep, missed those ones --> f0e369a

Reply:
I also wonder whether things like wheel and pip should appear in the dependencies.

Reply:
> Ah yep, missed those ones --> [f0e369a](https://github.com/zfit/phasespace/commit/f0e369a02ea995af29d61da3c718c83e409856ca)

I think this came from the `setup-cfg-fmt` hook, which unpacks that `$(test)s` reference

Reply:
Could be listed under `setup_requires`, or alternatively in the `pyproject.toml`, like setuptools-scm does:
https://pypi.org/project/setuptools-scm
I'm not sure though where/whether pip should be listed somewhere

Reply:
Ah yes, oc, wheel is already there, pip is not needed. Loosing oversight of the projects :D